sqlobject/src/main/java/org/jdbi/v3/sqlobject/customizer/internal/BindListFactory.java

```diff
@@ -49,10 +48,14 @@ public SqlStatementParameterCustomizer createForParameter(Annotation annotation,
                 case NULL:
                     stmt.define(name, "null");
                     return;
+                case JAVA_NULL:
+                    stmt.define(name, null);
```

Comment:
We could consider simply no-op'ing right here.

Reply:
(And then the naming question is simpler, we could simply call it `NONE` or `NOOP` or something like that)

Reply:
I'd rather not because that makes it rely on circumstance instead of explicit behavior